pkg/apis/serving/v1alpha2/framework_tensorflow.go

```diff
@@ -50,12 +53,24 @@ func (t *TensorflowSpec) GetContainer(modelName string, parallelism int, config
 		"--model_base_path=" + constants.DefaultModelLocalMountPath,
 	}
 
+	// Define TensorFlow Serving gRPC port.
+	TensorflowServingGRPCPortInt, err := strconv.Atoi(TensorflowServingGRPCPort)
+	if err == nil {
+		klog.Error(err)
+	}
+	port := []v1.ContainerPort{{Name: "h2c", ContainerPort: int32(TensorflowServingGRPCPortInt)}}
```

Comment:
I think it reads better if we default to nil here and then check gRPC

Reply:
thanks for the suggestions